plugins/CorePluginsAdmin/templates/macros.twig

```diff
@@ -141,7 +141,7 @@
                         </div>
                         <div class="plugin-desc-text">
 
-                            {{ plugin.info.description|raw|nl2br }}
+                            <div ng-bind-html="{{ plugin.info.description|raw|nl2br|json_encode|e('html_attr') }}"></div>
```

Comment:
It seems the only plugin that uses html is custom variables? I'd be tempted to simply not allow HTML in the plugin description.

Reply:
Don't have an opinion personally.

Reply:
I'll remove the HTML from custom variables and no longer allow HTML there.